pkg/report/linux.go

```diff
@@ -156,11 +157,31 @@ func (ctx *linux) Parse(output []byte) *Report {
 	if oops == nil {
 		return nil
 	}
-	title, report, format := extractDescription(ctx.extractConsoleOutput(output[rep.StartPos:]), oops)
```

Comment:
Drop extractConsoleOutput entirely. It must be dead code now.

Reply:
Done.